src/Components/Counter.jsx

```diff
@@ -1,15 +1,50 @@
-import React from 'react';
+import React, {useState} from 'react';
 import Display from './Display';
 import ButtonPad from './ButtonPad';
 
 
 function Counter() {
+    const [displayHistory, setDisplayHistory] = useState([]);
+    const [display,setDisplay] = useState(0);
+
+    const incrementCount = () => {
+        setDisplayHistory(displayHistory.concat(display + ', '));
+        setDisplay(display + 1);
+    };
+      
+    const decrementCount = () => {
+        setDisplayHistory(displayHistory.concat(display + ', '));
+        setDisplay(display - 1);
+    };
+      
+    const resetCount = () => {
+        setDisplayHistory([]);
+        setDisplay(0);
+    };
+
+    const doubleIncrementCount = () => {
+        setDisplayHistory(displayHistory.concat(display + ', '));
+        setDisplay(display + 2);
+    };
+
+    const fiveIncrementCount = () => {
+        setDisplayHistory(displayHistory.concat(display + ', '));
+        setDisplay(display + 5);
+    };
+
+    const tenIncrementCount = () => {
+        setDisplayHistory(displayHistory.concat(display + ', '));
```

Comment:
concat() works just fine.

[...displayHistory, display, ', ']
this is the newer way to do it, but either or should be fine